pkg/replicate/scheme.go

```diff
@@ -200,8 +197,8 @@ func (rs *replicationScheme) execute(ctx context.Context) error {
 
 // ensureBlockIsReplicated ensures that a block present in the origin bucket is
 // present in the target bucket.
-func (rs *replicationScheme) ensureBlockIsReplicated(ctx context.Context, id ulid.ULID) error {
-	blockID := id.String()
+func (rs *replicationScheme) ensureBlockIsReplicated(ctx context.Context, meta *metadata.Meta) error {
```

Comment:
Is this needed? ULID seemed enough? :thinking: 

Ensuring function interface is as small as possible gives usually more readable code (: (e.g you are sure that function does not modify / read other fields than ID)

Reply:
ah yea this can be reverted. I needed more data before.